bundles/org.openhab.binding.dirigera/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="dirigera" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>DIRIGERA Binding</name>
+	<description>IKEA Smarthome binding for DIRIGERA Gateway</description>
+	<connection>local</connection>
+
```

Comment:
If i'm not mistaken, the dirigera makes use of mDNS to anounce itself on the network. This can be used for the suggestion finder. You can look into the plugwiseha binding for an example. (combined with something like mdns explorer or other tool to show the mDNS announcements)

For reference: https://next.openhab.org/docs/developer/addons/addon.html#discovery-methods
Example: https://next.openhab.org/docs/developer/addons/addon.html#example

Reply:
True, pretty neat. So I'll remove the manual scanning and rely on mDNS

Reply:
mDNS discovery introduced